src/util/logging.ts

```diff
@@ -0,0 +1,32 @@
+import { notifications } from '@mantine/notifications';
+
+import { LogLevel } from '@/types/enums';
+
+export function showNotification(
+  title: string,
+  message: React.ReactNode,
+  level: LogLevel
+) {
+  const color = {
+    [LogLevel.Info]: 'white',
+    [LogLevel.Warning]: 'yellow',
+    [LogLevel.Error]: 'red'
```

Comment:
Place the colors in enums?

Personally i would prefer to defined the object first and then accessing with the key. For some reason it took me a while to realize that was what was happening :) Same below for log

Reply:
Yeah agree with the accessing, fixing that. Not sure if it's necessary to put the colors in enums, though?

Reply:
Up to you 🤷

Reply:
Agreeing with the comment on accessing here. This syntax is quite confusing